### PR TITLE
[Swift5] escape reserved words

### PR DESCRIPTION
**Pull request by aymanbagabas**
Swift reserved words should be escaped using backticks `` as the reference explains https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID412

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `5.1.x`, `6.0.x`
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

**Comment by 4brunu** [comment]
This is a nice improvement, but this is a breaking change, so we need to create a flag that keep's the old behaviour by default.

**Comment by aymanbagabas** [comment]
> This is a nice improvement, but this is a breaking change, so we need to create a flag that keep's the old behaviour by default.

I think this should be the default behavior but I understand that this would introduce a breaking change. Thus, I think it's best to merge this to the `5.2.x` branch instead of master.

**Comment by 4brunu** [comment]
For this to be merged in the branch 5.2.x it new to have a fallback to the old behaviour.
To do what you suggest, we need to merge this to 6.x.x.

**Comment by aymanbagabas** [comment]
> For this to be merged in the branch 5.2.x it new to have a fallback to the old behaviour.
> To do what you suggest, we need to merge this to 6.x.x.
> Check compatibility in the readme for more info.
> https://github.com/OpenAPITools/openapi-generator#11---compatibility

Will the changes from the other PRs be merged to `6.0.x` if they get merged to `master`?

**Comment by 4brunu** [comment]
Yes, but the version 6.0.x will only be release to the end of the year, or begin of next year.
So we should only consider that as last resort in case we can't find a way to make this backwards compatible with a switch.
But since this is a small change, we can create a switch to turn this on or off.

**Comment by 4brunu** [comment]
@aymanbagabas thanks for the changes.
Looks good to me.
The failure in CI is not related to the PR.
@wing328 for me this PR is ready to merge 👍

File: samples/client/petstore/swift5/rxswiftLibrary/docs/Model200Response.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **name** | **Int** |  | [optional] 
-**_class** | **String** |  | [optional] 
+**&#x60;class&#x60;** | **String** |  | [optional]
```
Comment (aymanbagabas) [inline samples/client/petstore/swift5/rxswiftLibrary/docs/Model200Response.md:7]: I should update the generated docs

**Comment by wing328** [comment]
@aymanbagabas please resolve the merge conflicts when you've time.

**Comment by aymanbagabas** [comment]
> @aymanbagabas please resolve the merge conflicts when you've time.

Resolved

**Pull request merged**